test/scaladoc/run/doc-source-url.scala

```diff
@@ -26,7 +26,7 @@ object Test extends ScaladocModelTest {
 
   override def model: Option[Universe] = newDocFactory.makeUniverse(Left(List(resourceFile)))
 
-  def scaladocSettings = "-doc-source-url file:€{FILE_PATH}@@€{FILE_EXT}@@€{FILE_PATH_EXT}@@€{FILE_LINE}"
+  def scaladocSettings = "-doc-source-url file:€{FILE_PATH}@@€{FILE_EXT}@@€{FILE_PATH_EXT}@@€{FILE_LINE} -sourcepath ."
```

Comment:
On the ticket, I asked why I had to use `-sourcepath` to show the problem but the reporters did not (in their examples).

Was there different behavior for `Paths.get("")` on different platforms? I saw in this fix that sourcepath is used if nonempty. I haven't looked at this test yet, but somehow I haven't nailed down why it's needed here.

Reply:
I think:
  - 2.12.11: `FILE_PATH` was `test/scaladoc/...` because partest a relative path to scaladoc
  - 2.12.12: `FILE_PATH` was changed to be relative to `Paths.get("")`, cwd is the root directory, so in the end it's unchanged
  - with this PR: if `-sourcepath` is not specified `FILE_PATH` is changed to be absolute